src/main/java/com/google/sps/tests/TestServlet.java

```diff
@@ -0,0 +1,67 @@
+package testing.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * This servlet is just to test a query on the Mintern database, and can be used as reference
+ * for later use.
+ */
+@WebServlet("/test-user")
```

Comment:
this is just from the merge of the test branch right?

Reply:
No. I did this before I knew the changes to the database were reflected on the terminal, so to test those changes, I created this servlet. These test servlets could give you information quicker jujst by typing the link in the URL, but are not essential. Now that I know I can use MySQL in the terminal, I don't know if they remain very practical, but Andrés did think they were good idea. I can delete it if you want.